enhancements/api-review/custom-tags-aws.md

```diff
@@ -62,22 +62,21 @@ list of user-defined tags to the OpenShift Installer, and everything created by
 bootstrapped components will apply those tags to all resources created in AWS, for the life of the cluster, and where supported by AWS.
 - Tags must be applied at creation time, in an atomic operation. It isn't acceptable to create an object and,
 after some period of time, apply the tags post-creation.
+- Tags can be updated day2 on the AWS resources.
 
 ### Non-Goals
 
-- To reduce initial scope, tags are applied only at creation time and not reconciled. If an administrator manually
-changes the tags stored in the infrastructure resource, behavior is undefined. See below.
-- To reduce initial scope, we are not implementing this for clouds other than AWS. We will not take any actions
+- To reduce initial scope, in case of standalone OpenShift, tags are applied only at creation time and not reconciled. If an administrator manually
+changes the tags stored in the infrastructure resource, behavior is undefined. 
+- To reduce initial scope, for hosted control plane deployments, we are not implementing this for clouds other than AWS. We will not take any actions
 to prohibit that later.
 
 ## Proposal
 
-### Existing design details
-
-New `experimentalPropagateUserTags` field added to `.platform.aws` of install config to indicate that the user tags should be applied to AWS
+New `propagateUserTags` field added to `.platform.aws` of install config to indicate that the user tags should be applied to AWS
```

Comment:
Where is the field stored **after** installation? In other words, how a CSI driver  operator knows it should sync tags or not?

Reply:
The field is applied in install-config while generating the infrastructure resource. There are no control explicit control fields proposed. Hence, it is enabled by default.